bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -554,6 +539,17 @@ private string GetName(IEnumerable<Claim> claims)
             return null;
         }
 
+        private SsoUser CreateSsoUserObject(string providerUserId, Guid userId, Guid orgId)
```

Comment:
Why not have this method literally create the record and persist it (making it async)? That will extend the re-use even more so.

Reply:
Updated - helper method now persists the `sso user` creation